addons/l10n_fr_fec/i18n/fr.po

```diff
@@ -236,18 +239,15 @@ msgstr ""
 #. module: l10n_fr_fec
 #: model:ir.model.fields.selection,name:l10n_fr_fec.selection__account_fr_fec__export_type__nonofficial
 msgid "Non-official FEC report (posted and unposted entries)"
-msgstr "Rapport FEC non-officiel (avec à la fois les entrées comptabilisées et non comptabilisées)"
+msgstr ""
+"Rapport FEC non-officiel (avec à la fois les entrées comptabilisées et non "
+"comptabilisées)"
 
 #: code:addons/l10n_fr_fec/wizard/fec.py:98
 #, python-format
 msgid "Invalid VAT number for company %s"
 msgstr "Numéro de TVA invalide sur la société %s"
 
-#. module: l10n_fr_fec
-#: view:account.fr.fec:0
```

Comment:
this terms is the duplicate one with line 59 which makes the update raising.

This fix as been done by poedit opening and saving the file !